3-networks-hub-and-spoke/envs/shared/interconnect.tf.example

```diff
@@ -17,7 +17,7 @@
 module "dns_hub_interconnect" {
   source = "../../modules/dedicated_interconnect"
 
-  vpc_name                = "net-dns"
+  vpc_name                = "vpc-net-dns"
```

Comment:
the infrastructure created by the module `"dns_hub_interconnect` is not necessary anymore.